packages/liveblocks-react/src/__tests__/useInboxNotifications.test.tsx

```diff
@@ -547,12 +552,12 @@ describe("useInboxNotifications: polling", () => {
     await waitFor(() => expect(pollerCount).toBe(0));
 
     // Wait for the first polling to occur after the initial render
-    jest.advanceTimersByTime(POLLING_INTERVAL);
+    jest.advanceTimersByTime(60_000);
     expect(initialCount).toBe(1);
     await waitFor(() => expect(pollerCount).toBe(1));
 
     // Advance time to simulate the polling interval
-    jest.advanceTimersByTime(POLLING_INTERVAL);
+    jest.advanceTimersByTime(60_000);
```

Comment:
Let's avoid relying on runtime constants in unit tests. If we change the constant, I would want these tests to fail.